mindmaps-migration/owl/src/main/java/io/mindmaps/migration/owl/OwlMindmapsGraphStoringVisitor.java

```diff
@@ -180,11 +180,16 @@ public Concept visit(OWLSubObjectPropertyOfAxiom axiom) {
         RelationType subRelation = migrator.relation(axiom.getSubProperty().asOWLObjectProperty());
         RelationType superRelation = migrator.relation(axiom.getSuperProperty().asOWLObjectProperty());
 
-        Map<String, String> roleMap = new HashMap<>();
-        roleMap.put(migrator.namer().subjectRole(superRelation.getId()), migrator.namer().subjectRole(subRelation.getId()));
-        roleMap.put(migrator.namer().objectRole(superRelation.getId()), migrator.namer().objectRole(subRelation.getId()));
-        createSubPropertyRule("sub-" + superRelation.getId() + UUID.randomUUID().toString(), superRelation, subRelation, roleMap, migrator.graph());
+        subRelation.superType(superRelation);
+        return null;
```

Comment:
This method never seems to anything other than null ? Maybe it should not have a return type.